Enterprise-CMCS/macpro-onemac
# add FAQ section and link from triage landing pages

Story: https://qmacbis.atlassian.net/browse/OY2-24094
Endpoint: See github-actions bot comment

### Details

Add new FAQ section and link to section from triage landing pages

### Changes

- Added new FAQ section
- Added new Route for FAQ section
- Added link to FAQ Route on triage page

### Test Plan

1. Login as state submitter
2. Go thru triage to land on each external system landing page
3. verify link navigates to FAQ with section expanded
4. verify clicking link to document in FAQ downloads expected file
